src/scrabble.js

```diff
@@ -1,6 +1,63 @@
 class Scrabble {
   // Write your implementation here
 
+  constructor (wordValue) {
+    this.wordValue = wordValue
+    this.isDouble = false
+    this.isTriple = false
+    this.obj = {
+      value1: ['A', 'E', 'I', 'O', 'U', 'L', 'N', 'R', 'S', 'T'],
+      value2: ['D', 'G'],
+      value3: ['B', 'C', 'M', 'P'],
+      value4: ['F', 'H', 'V', 'W', 'Y'],
+      value5: ['K'],
+      value8: ['J', 'X'],
+      value10: ['Q', 'Z']
+    }
+  }
+
+  _validateString (word) {
+    if (typeof word !== 'string' || word.length === 0) {
+      return false
+    } else return true
+  }
+
+  score () {
+    if (!this._validateString(this.wordValue)) return 0
```

Comment:
!this - means not this?

Reply:
Nice this returns value zero for invalid strings like empty ones.